client/packages/android/README.MD

```diff
@@ -143,12 +143,12 @@ Alternatively when using an emulator, navigate to `data/user/0/org.openmsupply.c
 
 #### Java
 
-If you get Java Compatibility* error in Android studio during gradle sync, you can download a new JDK via settings/preferences -> Build, Execution, Deployment -> Build Tools -> Gradle -> (drop down Gradle JDK:), and choose Java version as per [capacitor.build.gradle](https://github.com/openmsupply/open-msupply/blob/7befca66e757570bcec1eacc58099393e8eba8d0/client/packages/android/app/capacitor.build.gradle#L5). Then sync gradle files vai elephant icon on top right or File -> Sync Project with Gradle Files. 
+If you get Java Compatibility\* error in Android studio during gradle sync, you can download a new JDK via settings/preferences -> Build, Execution, Deployment -> Build Tools -> Gradle -> (drop down Gradle JDK:), and choose Java version as per [capacitor.build.gradle](https://github.com/openmsupply/open-msupply/blob/7befca66e757570bcec1eacc58099393e8eba8d0/client/packages/android/app/capacitor.build.gradle#L5). Then sync gradle files vai elephant icon on top right or File -> Sync Project with Gradle Files.
```

Comment:
via?

Reply:
You mean the *? don't know why its there, I removed it

Reply:
I mean `vai elephant`? What does that mean?

Reply:
ah ok "via the elephant"